src/TwigComponent/doc/index.rst

```diff
@@ -929,6 +929,10 @@ component's template:
       My Component!
     </div>
 
+.. note::
+
+    Quick reminder that passing an array of attributes to the *Component* requires the spread operator syntax (``<twig:MyComponent {{ ...attributes }} />``). See `Component HTML Syntax`_.
+
```

Comment:
Hmm in fact this may be confusing. Using the  `attributes` variable here may be missunderstood as we are talking about the component attributes just after.

What could be done here is a reminder link targetting [Passing & Rendering Attributes](https://symfony.com/bundles/ux-twig-component/current/index.html#passing-rendering-attributes) in the first pargraph of [Component Attributes](https://symfony.com/bundles/ux-twig-component/current/index.html#component-attributes)